### DISCUSSION TITLE
Inaccurate kinetic energy in system with a custom force?

### DISCUSSION THREAD

**Discussion by ajw2023** [glotzerlab/hoomd-blue#1754] [Support] [answered]
I'm a student in Erin Teich's group at Wellesley College, and I've run into an issue while trying to implement custom force. I have two versions of a Brownian simulation, one with custom force and one without. The custom force is set to an integer (as it is in hoomd's documentation example) and should produce a higher kinetic energy. When visualized in OVITO, the custom force uniformly pushes the particles in one direction. However, graphs of KE vs time for the two simulations are completely identical.  Any suggestions as to what could be causing this?

**Comment by joaander** [comment, marked as answer]
This is described [in the documentation](https://hoomd-blue.readthedocs.io/en/v4.6.0/module-md-methods.html#hoomd.md.methods.Brownian):
> In Brownian dynamics, particle velocities and angular momenta are completely decoupled from positions. At each time step, [Brownian](https://hoomd-blue.readthedocs.io/en/v4.6.0/module-md-methods.html#hoomd.md.methods.Brownian) draws a new velocity distribution consistent with the current set temperature so that [hoomd.md.compute.ThermodynamicQuantities](https://hoomd-blue.readthedocs.io/en/v4.6.0/module-md-compute.html#hoomd.md.compute.ThermodynamicQuantities) will report appropriate temperatures and pressures when logged or used by other methods.

Therefore, the kinetic energy reported by HOOMD-blue will always be thermally distributed on particles integrated by `Brownian`.
